ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/EthProtocolConfiguration.java

```diff
@@ -14,169 +14,16 @@
-    return maxGetBlockHeaders == that.maxGetBlockHeaders
-        && maxGetBlockBodies == that.maxGetBlockBodies
-        && maxGetReceipts == that.maxGetReceipts
-        && maxGetNodeData == that.maxGetNodeData
-        && maxGetPooledTransactions == that.maxGetPooledTransactions;
-  }
-
-  @Override
-  public int hashCode() {
-    return Objects.hash(maxGetBlockHeaders, maxGetBlockBodies, maxGetReceipts, maxGetNodeData);
-  }
-
-  @Override
-  public String toString() {
-    return MoreObjects.toStringHelper(this)
-        .add("maxGetBlockHeaders", maxGetBlockHeaders)
-        .add("maxGetBlockBodies", maxGetBlockBodies)
-        .add("maxGetReceipts", maxGetReceipts)
-        .add("maxGetNodeData", maxGetNodeData)
-        .add("maxGetPooledTransactions", maxGetPooledTransactions)
-        .toString();
-  }
-
-  public static class Builder {
-    private PositiveNumber maxGetBlockHeaders =
-        PositiveNumber.fromInt(EthProtocolConfiguration.DEFAULT_MAX_GET_BLOCK_HEADERS);
-
-    private PositiveNumber maxGetBlockBodies =
-        PositiveNumber.fromInt(EthProtocolConfiguration.DEFAULT_MAX_GET_BLOCK_BODIES);
-
-    private PositiveNumber maxGetReceipts =
-        PositiveNumber.fromInt(EthProtocolConfiguration.DEFAULT_MAX_GET_RECEIPTS);
-
-    private PositiveNumber maxGetNodeData =
-        PositiveNumber.fromInt(EthProtocolConfiguration.DEFAULT_MAX_GET_NODE_DATA);
-
-    private PositiveNumber maxGetPooledTransactions =
-        PositiveNumber.fromInt(EthProtocolConfiguration.DEFAULT_MAX_GET_POOLED_TRANSACTIONS);
-
-    private boolean eth65Enabled = EthProtocolConfiguration.DEFAULT_ETH_65_ENABLED;
-
-    public Builder maxGetBlockHeaders(final PositiveNumber maxGetBlockHeaders) {
-      this.maxGetBlockHeaders = maxGetBlockHeaders;
-      return this;
-    }
-
-    public Builder maxGetBlockBodies(final PositiveNumber maxGetBlockBodies) {
-      this.maxGetBlockBodies = maxGetBlockBodies;
-      return this;
-    }
-
-    public Builder maxGetReceipts(final PositiveNumber maxGetReceipts) {
-      this.maxGetReceipts = maxGetReceipts;
-      return this;
-    }
-
-    public Builder maxGetNodeData(final PositiveNumber maxGetNodeData) {
-      this.maxGetNodeData = maxGetNodeData;
-      return this;
-    }
-
-    public Builder maxGetPooledTransactions(final PositiveNumber maxGetPooledTransactions) {
-      this.maxGetPooledTransactions = maxGetPooledTransactions;
-      return this;
-    }
-
-    public Builder eth65Enabled(final boolean eth65Enabled) {
-      this.eth65Enabled = eth65Enabled;
-      return this;
-    }
-
-    public EthProtocolConfiguration build() {
-      return new EthProtocolConfiguration(
-          maxGetBlockHeaders.getValue(),
-          maxGetBlockBodies.getValue(),
-          maxGetReceipts.getValue(),
-          maxGetNodeData.getValue(),
-          maxGetPooledTransactions.getValue(),
-          eth65Enabled);
-    }
-  }
+  @Builder.Default Integer maxGetBlockHeaders = 192;
```

Comment:
Why the boxed type?  Should it just be `int`?

Reply:
Yeah i will use the primitive type instead

Reply:
Done